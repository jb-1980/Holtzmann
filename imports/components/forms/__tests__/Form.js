import renderer from "react-test-renderer";
import Form from "../Form";

it("should exist", () => {
  const form = renderer.create(<Form />);
  expect(form).toMatchSnapshot();
});

it ('accepts all props', () => {
  const fn = jest.fn();
  const form = renderer.create(<Form id="test1" submit={fn} classes="test3" action="hello" method="test5" style={{display: "block"}} fieldsetTheme="test7"/>);

  expect(form).toMatchSnapshot();
});
