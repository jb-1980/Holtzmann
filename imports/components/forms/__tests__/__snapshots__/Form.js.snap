exports[`test accepts all props 1`] = `
<form
  action="hello"
  className="hard-ends test3"
  id="test1"
  method="test5"
  onSubmit={[Function anonymous]}
  style={
    Object {
      "display": "block"
    }
  }>
  <fieldset
    className="test7" />
</form>
`;

exports[`test should exist 1`] = `
<form
  action={undefined}
  className="hard-ends"
  id={undefined}
  method={undefined}
  onSubmit={undefined}
  style={undefined}>
  <fieldset
    className="flush-bottom" />
</form>
`;
